mantidimaging/gui/windows/main/model.py

```diff
@@ -86,15 +88,18 @@ def do_images_saving(self, images_id: uuid.UUID, output_dir: str, name_prefix: s
                                      pixel_depth=pixel_depth,
                                      progress=progress)
         images.filenames = filenames
+        logger.info(f"Export completed. Files saved: {filenames}")
```

Comment:
Lets not include all the filenames. It could be
`logger.info(f"Export completed. Files saved: {filenames[:2]} ...")`
so that it only shows the first few.